lib/index.js

```diff
@@ -17,6 +17,7 @@ export * from './CurrencyExchangeRateFields';
 export * from './CurrencySymbol';
 export * from './DeleteHoldingsModal';
 export * from './Donors';
+export * from './DonorFilter';
```

Comment:
This is the filter by several donors, the correct name would probably be "Donor**s**Filter"